sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update_slm.hpp

```diff
@@ -593,34 +593,17 @@ template <int N, template <class, int> class Op, bool UseMask,
           TestFeatures Features, bool UseAcc>
 bool test_fp_types(queue q) {
   bool passed = true;
-  if constexpr (Features == TestFeatures::DG2 ||
```

Comment:
I removed this constexpr check for DG2/PVC because it is already checked before calling any FP ops (i.e. before test_fp_types_and_sizes()).